src/Microsoft.TemplateEngine.Edge/Settings/SettingsLoader.cs

```diff
@@ -268,6 +281,14 @@ private IEnumerable<MountPointInfo> FindMountPointsToScan(bool forceRebuild)
                     // TODO: This should never happen - throw an error?
                     continue;
                 }
+
```

Comment:
Note for reviewers: the fix itself is 7 lines below
The rest is needed to make it testable